pnpm-workspace.yaml

```diff
@@ -13,6 +13,7 @@ catalog:
   '@types/picomatch': ^4.0.2
   ansis: ^4.1.0
   bumpp: ^10.2.3
+  bun-types-no-globals: github:alii/bun-types-no-globals
```

Comment:
It should be published to npm.

Reply:
Cool - I'll do that tomorrow!

Reply:
Published - https://www.npmjs.com/package/bun-types-no-globals